### PR TITLE
Feature/clean and restructure

### PR DESCRIPTION
**Pull request by MonaHer**
I created tests for two components and
cleaned up the code: 
- deleted a lot of redundant code
- restructured components and pages
- created `styles.js` for the styled components of every component
- changed a lot of style 
- added responsive style
I know it became a pretty big PR (bigger as it was supposed to regarding the US) as I did a lot of refactoring and styling in many different pages and components (I'm sorry for that !:)) 
Pls review :)

### CODE REVIEW

File: components/Navigation/index.js [hunk 1]
```diff
@@ -1,80 +1,47 @@
...
     <>
-      <NavBar>
-        <StyledNavBarLink href={`/`} active={router.pathname === "/"}>
+      <StyledNavBar>
+        <StyledNavBarLink href={`/`}>
           <StyledNavBarIcon
             path={mdiShimmer}
             size={2}
-            active={router.pathname === "/"}
+            $isActive={router.pathname === "/"}
```
Comment (Kheirah) [inline components/Navigation/index.js:17]: I think you meant to say `isActive` without the dollar sign.

File: components/Navigation/index.js [hunk 2]
```diff
@@ -1,80 +1,47 @@
...
-        <StyledNavBarLink
-          href={`/artworks-collection`}
-          active={router.pathname === "/artworks-collection"}
-        >
+        <StyledNavBarLink href={`/artworks-collection`}>
           <StyledNavBarIcon
             path={mdiMagnify}
             size={2}
-            active={router.pathname === "/artworks-collection"}
+            $isActive={router.pathname === "/artworks-collection"}
```
Comment (Kheirah) [inline components/Navigation/index.js:25]: Same here, no dollar sign.

File: components/Navigation/index.js [hunk 3]
```diff
@@ -1,80 +1,47 @@
...
-        <StyledNavBarLink
-          href={`/my-notes`}
-          active={router.pathname === "/my-notes"}
-        >
+        <StyledNavBarLink href={`/my-notes`}>
           <StyledNavBarIcon
             path={mdiNotebookHeartOutline}
             size={2}
-            active={router.pathname === "/my-notes"}
+            $isActive={router.pathname === "/my-notes"}
```
Comment (Kheirah) [inline components/Navigation/index.js:33]: And here.

File: components/Navigation/index.js [hunk 4]
```diff
@@ -1,80 +1,47 @@
...
-        <StyledNavBarLink
-          href={`/favorites`}
-          active={router.pathname === "/favorites"}
-        >
+        <StyledNavBarLink href={`/favorites`}>
           <StyledNavBarIcon
             path={mdiCardsHeart}
             size={2}
-            active={router.pathname === "/favorites"}
+            $isActive={router.pathname === "/favorites"}
```
Comment (Kheirah) [inline components/Navigation/index.js:40]: And here.

File: components/Navigation/styles.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+export const StyledNavBarLink = styled(Link)`
+  width: 25%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const StyledNavBarIcon = styled(Icon)`
+  color: ${({ isActive }) => (isActive ? "hotpink" : "whitesmoke")};
```
Comment (Kheirah) [inline components/Navigation/styles.js]: `isActive` is spelled without a dollar sign, which is why you need to use the prop by the same name. Either include the dollar sign here or remove it everywhere else.

**Review by Kheirah — CHANGES_REQUESTED** [review]
Looking good!

Just a minor note on a prop, have a look below.

File: components/Navigation/styles.js (hunk 5 above)
Comment (MonaHer) [reply components/Navigation/styles.js]: Thank you very much for your review! :)

**Review by Kheirah — APPROVED** [review]
👍🏼

**Pull request merged**